src/bindings/python/setup.cfg

```diff
@@ -33,16 +33,28 @@ deps = -rrequirements.txt
 
 [flake8]
 # ignore:
+# A001 variable "var_name" is shadowing a python builtin
+# A002 argument "..." is shadowing a python builtin
 # D100 - Missing docstring in public module
+# D101 - Missing docstring in public class
 # D102 - Missing docstring in public method
 # D103 - Missing docstring in public function
 # D104 - Missing docstring in public package
 # D105 - Missing docstring in magic method
 # D107 - Missing docstring in __init__
 # D412 - No blank lines allowed between a section header and its content
 # F401 - module imported but unused
+# N803 - argument name '...' should be lowercase
+# T001 - print found
 # W503 - line break before binary operator (prefer line breaks before op, not after)
-ignore=D100,D102,D103,D104,D105,D107,D412,F401,W503
+# RST301 - Unexpected indentation
+# TAE002 - too complex annotation
+# PT007 - wrong values type @pytest.mark.parametrize, expected list of tuples
+# PT012 -  pytest.raises() block should contain a single simple statement
+# VNE001 - single letter variable names like 'X' are not allowed
```

Comment:
please, keep these error about variable names just for [`src/openvino/runtime/opset1`, `src/openvino/runtime/opset5`].
You ignored for all files but I'd prefer that new files will be more style-correct.